### PR TITLE
subwallet updates

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR applies the updates to the SubWallet page from @siddankthep and merges the browser extension and mobile app sections to avoid duplicate content.

⚠️ Need to circle back to the checklist for translations and after translation requirements (translations are underway already and images have moved) ⚠️ 

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Comment by eshaben** [comment]
@siddankthep, do you want to take a look at the updates to make sure they're all ok?

**Comment by siddankthep** [comment]
@eshaben thanks for the review! Only one thing that I would like to change. On line 20 "There are several ways you can interact with SubWallet: they have a browser extension, a mobile app, and a web-accessible dashboard.", can you make all of the platform bold-faced? 
-> "There are several ways you can interact with SubWallet: they have a **browser extension, a mobile app, and a web-accessible dashboard.**"

**Comment by siddankthep** [comment]
@eshaben Also, SubWallet also supports staking on and connecting to Moonbeam dApp (the extension allows connecting directly, the mobile app and web dashboard allows connecting through WalletConnect V2). Are there any page that we can add the guide for connecting and staking? Or should we add them directly onto this page?

**Comment by eshaben** [comment]
@siddankthep Thanks for taking the time to review this! I put the list of platforms in bold.

As far as the Moonbeam dApp-related content, we don't have any documentation for the dApp on the developer docs. Currently, our how-to-stake guides for the dApp are on the moonbeam.network site. There are some major changes to the network site planned for Q1, so I'll make the requests internally to add this to the list of tutorials once these changes have been implemented.

File: tokens/connect/subwallet.md [hunk 1]
```diff
@@ -59,90 +67,142 @@ You'll then be prompted to back up your seed phrase. This is an important step,
...
-- Sending a token transfer to another address
-- Adding ERC-20s to Metamask and interacting with them
-- Adding ERC-721s to Metamask and interacting with them
+- Receiving a token from another address
+- Sending a token to another address
+- Adding tokens to SubWallet and interacting with them
+
+### Receive a Token {: #receive-a-token }
+
+To receive a token from another account, you would need to show your wallet address to your counter-party, and they can send their assets to such address.
```
Comment (themacexpert) [inline tokens/connect/subwallet.md:157]:
```suggestion
To receive a token from another account, you would need to show your wallet address to your counterparty, and they can send their assets to such address.
```

**Review by themacexpert — APPROVED** [review]
LGTM

**Pull request merged**